shared/productTypes.ts

```diff
@@ -445,6 +447,46 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
 			productFilenamePart: 'Newspaper - Home Delivery',
 		},
 	},
+	nationaldelivery: {
+		productTitle: calculateProductTitle('Newspaper Delivery'),
+		friendlyName: () => 'newspaper national delivery subscription',
+		shortFriendlyName: 'newspaper national delivery',
+		productType: 'nationaldelivery',
+		groupedProductType: 'subscriptions',
+		allProductsProductTypeFilterString: 'HomeDelivery', // Is this right?
+		urlPart: 'homedelivery', // Is this right?
```

Comment:
I think probably this should be unique, because ReactRouter is going to pick one (not sure which? maybe the first) and if we specify different config options than for normal home delivery it could create problems

Reply:
see `MMAPage.tsx`